.env-example

```diff
@@ -0,0 +1 @@
+MONGODB_URI=mongodb://localhost:27017/basic-auth
```

Comment:
Be sure to add your `.env` to your `.gitignore`! 

You don't want to push your .env as it could contain secure information.